src/crypto/index.js

```diff
@@ -411,12 +413,13 @@ Crypto.prototype.bootstrapSecretStorage = async function({
                 );
             }
         } else {
+            if (setupNewSecretStorage) throw new Error("What the HELL");
```

Comment:
probably want to remove this debug line

Reply:
WHOOPS

Reply:
I really need some kind of pre-push hook I can use in conjunction with some magic string to yell at me for this